src/util/color.h

```diff
@@ -0,0 +1,47 @@
+/*
+Minetest
+Copyright (C) 2016 juhdanad, Dániel Juhász <juhdanad@gmail.com>
+
+This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation; either version 2.1 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License along
+with this program; if not, write to the Free Software Foundation, Inc.,
+51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+#ifndef SRC_UTIL_COLOR_H_
+#define SRC_UTIL_COLOR_H_
+
+#include "irrlichttypes.h"
+
+//! A color represented in three unsigned chars.
+struct Color {
```

Comment:
video::SColor works with one u32. If you read for example `SColor::setRed(u32)`, it makes a lot of bit operations.
This struct also does not have alpha.